src/main/java/core/basesyntax/UpdateStorage.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface UpdateStorage {
```

Comment:
"StorageUpdater" sounds better

Reply:
done

Reply:
i don't see any changes in code, probably, you haven't pushed them